test/game_stats_test.rb

```diff
@@ -56,8 +56,8 @@ def test_it_can_calculate_average_goals_per_game
   end
 
   def test_it_can_calculate_average_goals_per_season
-    goals_by_season = { '20152016' => 5.33, '20132014' => 5, '20142015' => 3,
-                        '20162017' => 4}
+    goals_by_season = { 20152016 => 5.33, 20132014 => 5, 20142015 => 3,
```

Comment:
good job on making this numbers instead of strings

Reply:
Thanks!